proposed/http-client/http-client-meta.md

```diff
@@ -5,15 +5,15 @@ HTTP Client Meta Document
 
 HTTP requests and responses are the two fundamental objects in web programming.
 All clients communicating to an external API use some form of HTTP client. Many
-libraries are coupled to one specific client or implement a client and/or adapter
-layer themselves. This leads to bad library design, version conflicts, or too much
-code not related to the library domain.
+libraries are coupled to one specific client or implement a client and/or
+adapter layer themselves. This leads to bad library design, version conflicts,
+or code unrelated to the library domain.
 
 ## Why bother?
 
 Thanks to PSR-7 we know how HTTP requests and responses ideally look, but nothing
 defines how a request should be sent and a response received. A common interface for HTTP
-clients will allow libraries to be decoupled from specific implementation such as Guzzle.
+clients will allow libraries to be decoupled from specific implementations.
```

Comment:
We don't know if Guzzle will continue to be well-known in the future. As such, removed the reference here, as the important bit is decoupling from implementations.

Reply:
thanks! this one bothered me as well but i felt it might help giving people concrete ideas. but you are right about future proof, and additionally it could be seen as random hate on guzzle (which its not of course, mark is even maintaining guzzle atm)